app/src/main/kotlin/com/exactpro/th2/lwdataprovider/http/GetOneEvent.kt

```diff
@@ -84,7 +86,14 @@ class GetOneEvent(
 
         logger.info { "Received get event request ($eventId)" }
 
-        val reqContext = HttpGenericResponseHandler(queue, sseResponseBuilder, Event::eventId, SseResponseBuilder::build)
+        val reqContext = HttpGenericResponseHandler(
+            queue,
+            sseResponseBuilder,
+            convExecutor,
```

Comment:
thought: I don't think it makes sense to use a separate thread for a `get` request. We occupy two threads instead of one (converter thread + HTTP thread). It would be better to replace it with an `in place` executor (`Runnable::run` should probably work)